src/cc/arduino/plugins/wifi101/flashers/java/SSLCertDownloader.java

```diff
@@ -82,7 +82,7 @@ public void checkServerTrusted(X509Certificate[] xcs, String string) throws Cert
 		connection.setSSLSocketFactory(ssl.getSocketFactory());
 
 		Certificate[] certificates = null;
-		if (connection.getResponseCode() == 200) {
+		if (connection.getResponseCode() > 0) {
```

Comment:
When `getResponseCode()` returns `0`? Maybe we can get rid of it completely?
Basically, any respone from the server should be good.